config/samples/jaegertracing.io_v1_jaeger.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "my-jaeger"
+spec:
+  strategy: allInOne
+  allInOne:
+    image: jaegertracing/all-in-one:1.28
```

Comment:
this is not a good example. The image will get outdated soon :)

Reply:
Removed the image :)